README.md

```diff
@@ -0,0 +1,116 @@
+# **ALTERNATE BIT PROTOCOL USING CADMIUM**
+---
+### ORGANINIZATION
+---
+#### Carleton University
+### AUTHORS
+---
+**Originally Developed by**: Cristina Ruiz Martin<br/>
+**Modified by**: Peter Bliznyuk-Kvitko and Shubham Agrawal
```

Comment:
Hi,
I would like to suggest that, it will be really good if you make the reference links or hyperlinks at the names, which should take me to respective Author's GitHub profile.

Reply:
done